### DISCUSSION TITLE
GSOD

### DISCUSSION THREAD

**Discussion by Emidowojo** [fonttools/fonttools#3477] [Q&A] [answered]
Hello @n8willis please how can I join your slack channel or some contact information so we can discuss the project? 
My name is Emidowojo and I am a Technical writer who is interested in contributing to your project.
Thank you!

**Comment by areeshatariq** [comment]
Hello @n8willis. I am also looking forward to contributing to fonttools in GSoD 2024. Please guide on how to join slack channel or any discussion forum?

**Comment by behdad** [comment]
Hi all. Please see:

   https://github.com/fonttools/fonttools/wiki/Season-of-Docs-2024-project-proposal

We will post here more information as it becomes available.

**Comment by behdad** [comment, marked as answer]
Hi @Emidowojo and @areeshatariq,

I appreciate you expressing interest in improving the FontTools documentation. However, as our application clearly [says](https://github.com/fonttools/fonttools/wiki/Season-of-Docs-2024-project-proposal#scope):

> We have identified a technical writing candidate who has a proven track record for documentation projects, familiarity with the fontTools codebase, and a thorough understanding of font technology.

As such unfortunately we have to turn down your offer for help.
